### ISSUE TITLE
Accessibility Audit: Header and user menu drop-down

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5779]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the header and user menu dropdown. Do the page scan as well as any relevant guided tests.
Use cases to test:
* org with 1 facility, and multiple facilities
* view header and user menu as someone with test_only access, standard_user access, and admin access

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by BobanL** [comment]
No accessibility issues found.
